acme.sh

```diff
@@ -4851,6 +4851,7 @@ Parameters:
   --useragent                       Specifies the user agent string. it will be saved for future use too.
   --accountemail                    Specifies the account email for registering, Only valid for the '--install' command.
   --accountkey                      Specifies the account key path, Only valid for the '--install' command.
+  --apiurl, --server, -s [acme api URL]   Specifies the particular acme api endpoint to use
```

Comment:
remove `--apiurl` and `-s`,  just keep `--server` only.
And let's use the same descp as certbot:
```
--server SERVER       ACME Directory Resource URI. (default:
                        https://acme-v01.api.letsencrypt.org/directory)
```